jdaviz/configs/mosviz/tests/test_helper.py

```diff
@@ -355,3 +355,18 @@ def test_custom_columns(mosviz_app, image, spectrum1d, spectrum2d):
     assert "Redshift" in mosviz_app.get_column_names(True)
     mosviz_app.set_visible_columns()
     assert "Redshift" not in mosviz_app.get_column_names(True)
+
+
+@pytest.mark.filterwarnings('ignore')
+def test_redshift_column(mosviz_app, image, spectrum1d, spectrum2d):
+    spectra1d = [spectrum1d] * 2
+    spectra2d = [spectrum2d] * 2
+
+    mosviz_app.load_data(spectra1d, spectra2d, images=image)
+
+    mosviz_app.update_column("Redshift", 0.1, row=0)
+    assert list(mosviz_app.specviz.get_spectra().values())[0].redshift.value == pytest.approx(0.1)
```

Comment:
If redshift is a Quantity, you can use `assert_quantity_allclose` from `astropy`.

Reply:
Otherwise, `numpy.testing.assert_allclose` is more consistent with the rest of the test suite.